cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/TicketState.java

```diff
@@ -21,7 +23,7 @@
      *
      * @return the last time the ticket was used.
      */
-    long getLastTimeUsed();
+    ZonedDateTime getLastTimeUsed();
```

Comment:
Don't think this will work with JPA ticket registries. We'll likely run into either schema or serialization issues. Let's keep this as long.